frontend/src/cards/ui/SviAlert.tsx

```diff
@@ -4,6 +4,8 @@ import { MetricQueryResponse } from "../../data/query/MetricQuery";
 import { Fips } from "../../data/utils/Fips";
 import { urlMap } from "../../utils/externalUrls";
 import styles from "./SviAlert.module.scss";
+import { HashLink as Link } from "react-router-hash-link";
```

Comment:
What do you think about importing it as just `HashLink` ? Because use `Link` in other components and I anticipate using this for more interconnections in other components and it might be nice to just keep them obviously distinct?

Reply:
Makes perfect sense.